### PR TITLE
Time left readability bug "1 hr 60 min"

### LINKED ISSUE #190: Time left readability bug "1 hr 60 min"
Screen Shot 2021-07-02 at 5 12 57 PM
Noticed this little readability bug... should read "2 hours"

Haven't tested at larger time scales yet to find additional edge cases

### PR DESCRIPTION
**Pull request by corwintines**
Fixes: https://github.com/ethereum/clrfund/issues/190

Note: play around with deployTestRound times and redeploy round to make this easier to test with different times

### CODE REVIEW

File: vue-app/src/utils/dates.ts [hunk 1]
```diff
@@ -27,7 +27,11 @@ export function getTimeLeft(date: DateTime): TimeLeft {
     return { days: 0, hours: 0, minutes: 0 }
   }
   const { days, hours, minutes } = date.diff(now, ['days', 'hours', 'minutes'])
-  return { days, hours, minutes: Math.ceil(minutes) }
+  return {
+    days,
+    hours: days !== 0 ? hours : Math.ceil(minutes) % 60 ? hours : hours + 1, // If days are 0, and minutes are 60, add 1 to the hours to display properly
+    minutes: Math.ceil(minutes) % 60 ? Math.ceil(minutes) : 0, // If minutes are 60, carry that over to the hours above and show 0
+  }
```
Comment (pettinarip) [inline vue-app/src/utils/dates.ts]: Consider using luxon built-in format functions. The error in the original implementation is that we are using the absolute amounts of each `diff` unit without having context.
I would let all this be handled by luxon, like this:
```ts
const diff = date.diff(now);
const [days, hours, minutes] = diff.toFormat("d h m").split(' ').map(Number)

return {
  days,
  hours,
  minutes
};
```

Note: TBH, in an ideal scenario I would just return the raw diff and let the consumers handle the format they want but that is a bigger refactor I don't think it's necessary right now.

**Review by pettinarip — APPROVED** [review]

File: vue-app/src/utils/dates.ts (hunk 1 above)
Comment (corwintines) [reply vue-app/src/utils/dates.ts]: This makes sense, thanks. I didn't feel good about what I was doing, so glad you could provide a clear answer for that.
